<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MesonNewBuildDir</class>
 <widget class="QDialog" name="MesonNewBuildDir">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>670</width>
    <height>700</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="hl_buildDir">
     <item>
      <widget class="QLabel" name="l_buildDir">
       <property name="text">
        <string>Build directory:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="KUrlRequester" name="i_buildDir">
       <property name="mode">
        <set>KFile::Directory|KFile::ExistingOnly|KFile::LocalOnly</set>
       </property>
       <property name="placeholderText">
        <string>Full Path to the new build directory</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="MesonOptionsView" name="options" native="true"/>
   </item>
   <item>
    <widget class="MesonAdvancedSettings" name="advanced" native="true"/>
   </item>
   <item>
    <layout class="QHBoxLayout" name="hl_bottomBar">
     <item>
      <widget class="QLabel" name="l_statusMessage">
       <property name="text">
        <string>Status message...</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDialogButtonBox" name="b_buttonBox">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="standardButtons">
        <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok|QDialogButtonBox::Reset</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KUrlRequester</class>
   <extends>QWidget</extends>
   <header>kurlrequester.h</header>
  </customwidget>
  <customwidget>
   <class>MesonAdvancedSettings</class>
   <extends>QWidget</extends>
   <header>settings/mesonadvancedsettings.h</header>
   <container>1</container>
   <slots>
    <signal>configChanged()</signal>
   </slots>
  </customwidget>
  <customwidget>
   <class>MesonOptionsView</class>
   <extends>QWidget</extends>
   <header>settings/mesonoptionsview.h</header>
   <container>1</container>
   <slots>
    <signal>configChanged()</signal>
   </slots>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>b_buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>MesonNewBuildDir</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>564</x>
     <y>643</y>
    </hint>
    <hint type="destinationlabel">
     <x>329</x>
     <y>333</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>b_buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>MesonNewBuildDir</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>564</x>
     <y>643</y>
    </hint>
    <hint type="destinationlabel">
     <x>329</x>
     <y>333</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>advanced</sender>
   <signal>configChanged()</signal>
   <receiver>MesonNewBuildDir</receiver>
   <slot>updated()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>324</x>
     <y>571</y>
    </hint>
    <hint type="destinationlabel">
     <x>324</x>
     <y>312</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>options</sender>
   <signal>configChanged()</signal>
   <receiver>MesonNewBuildDir</receiver>
   <slot>updated()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>334</x>
     <y>263</y>
    </hint>
    <hint type="destinationlabel">
     <x>334</x>
     <y>349</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>i_buildDir</sender>
   <signal>textChanged(QString)</signal>
   <receiver>MesonNewBuildDir</receiver>
   <slot>updated()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>399</x>
     <y>89</y>
    </hint>
    <hint type="destinationlabel">
     <x>334</x>
     <y>349</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>resetFields()</slot>
  <slot>updated()</slot>
 </slots>
</ui>
